builders/interoperability/xcm/xc-registration/assets.md

```diff
@@ -37,195 +37,122 @@ If a channel between the chains already exists, you'll need to create a forum po
-- `--collectiveThreshold` or `--c` - (optional) - the number of council votes that are needed to approve the proposal. Defaults to `1`
-- `--at-block` - (optional) - the block number at which the call should get executed
-- `--track` - (optional) - the Track the proposal should go through for OpenGov proposals. For Moonbeam, the General Admin Origin should be used
-- `--delay` - (optional) - the delay period (in blocks) after a proposal has passed and before it can be executed. Defaults to `100` blocks
+Using the above information, you can generate the encoded call data for the `addAsset` call either via the Polkadot API or on [Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.api.moonbeam.network#/extrinsics){target=\_blank}.
 
-Altogether, you can use the following command to submit a preimage and proposal using OpenGov, which batches the asset registration and sets the asset's units per second.
+To create a batch transaction that combines both the `xcmWeightTrader.addAsset` and the `evmForeignAssets.createForeignAsset` calls together, you can use the [Polkadot API's Batch Method](/builders/substrate/libraries/polkadot-js-api/#batching-transactions){target=\_blank}. As mentioned previously, the [XCM Asset Registrator](https://github.com/Moonsong-Labs/xcm-tools/blob/main/scripts/xcm-asset-registrator.ts){target=\_blank} script can help you build and submit the required calls. 
 
-=== "Moonbeam"
+### Submit the Preimage and Proposal for Asset Registration {: #submit-preimage-proposal }
 
-    ```bash
-    yarn register-asset -w wss://wss.api.moonbeam.network  \
-    --asset 'INSERT_ASSET_MULTILOCATION' \
-    --symbol "INSERT_TOKEN_SYMBOL" \
-    --decimals INSERT_TOKEN_DECIMALS \
-    --name "INSERT_TOKEN_NAME" \
-    --units-per-second INSERT_UNITS_PER_SECOND \
-    --existential-deposit 1 \
-    --sufficient true \
-    --account-priv-key "0x5fb92d6e98884f76de468fa3f6278f8807c48bebc13595d45af5bdc4da702133" \
-    --send-preimage-hash true \
-    --send-proposal-as v2
-    --track '{ "Origins": "GeneralAdmin" }'
-    ```
+Your next task is to submit the preimage of your batched call containing both the `xcmWeightTrader.addAsset` and the `evmForeignAssets.createForeignAsset` by following the guidelines in the [Submit a Democracy Proposal Guide](/tokens/governance/proposals/#submitting-a-preimage-of-the-proposal){target=\_blank}.
 
-=== "Moonriver"
+For Moonbase Alpha, you do not need to go through governance, as Moonbase Alpha has `sudo` access. Instead, you can provide the output of the batch call data to the Moonbeam team, and the Moonbeam Team can submit the call with `sudo`. This will be a faster and easier process than going through governance. However, you may still wish to go through governance on Moonbase Alpha to prepare for Moonbeam's governance process.
```

Comment:
```suggestion
You do not need to go through governance for Moonbase Alpha, as Moonbase Alpha has sudo access. Instead, you can provide the output of the batch call data to the Moonbeam team, and they can submit the call with sudo. This will be a faster and easier process than going through governance. However, you may still wish to go through governance on Moonbase Alpha to prepare for Moonbeam's governance process.
```